packages/yourloops/pages/hcp/patients-list-bar.tsx

```diff
@@ -27,14 +27,20 @@
  */
 
 import * as React from "react";
-import { useHistory } from "react-router-dom";
 
+import { FilterType, Team } from "./types";
+import { REGEX_EMAIL, defer } from "../../lib/utils";
```

Comment:
Local imports at the last position is the best !

Reply:
Totally agreed, I forgot to setup my imports organizer